content/tutorials/portenta-h7/por-ard-lvgl/content.md

```diff
@@ -37,15 +37,12 @@ we are going to work on the **Portenta_lvgl** example , The Portenta H7 comes wi
    
    Once you have installed the library, create a new Sketch **File --> New**
    
-   *** If you cant find the sketch, make sure you have selected the right board inside Tools > Boards ***
+   *** Make sure you have selected the right board inside Tools > Boards ***
    
-   ![por_ard_lvgl_select_example](assets/por_ard_lvgl_select_example.svg)
    
    
    
-3. The Hello World label widget 
-
-   Lets start by including the libraries that we are going to use.
+3. Including the libraries that we are going to use.
```

Comment:
Shall we combine step 3 with 4 and have something like this 

3. Adding Buttons and Labels
Lets start by including the `Portenta_LittleVGL.h` and `USBHost.h` libraries.  
```cpp
 #include "Portenta_LittleVGL.h"
 #include "USBHost.h"
```
We will then create a button widget using the `lv_btn` object. This object also allows you to have an inner label whose text can be easily configured. The following lines of code initialises the button variable and its label, `myCustomLabel` which you can add it at the beginning of the sketch.

Reply:
You can then have a separate section for the `void setup()`part as step 4